pkg/gameservers/portallocator.go

```diff
@@ -106,7 +106,6 @@ func (pa *PortAllocator) Run(stop <-chan struct{}) error {
 }
 
 // Allocate assigns a port to the GameServer and returns it.
-// Return ErrPortNotFound if no port is allocatable
```

Comment:
This would be actually a warning:
```
11m         Warning   FailedScheduling          pod/simple-udp-qbx79-c2vll                           0/6 nodes are available: 2 node(s) had taints that the pod didn't tolerate, 4 node(s) didn't have free ports for the requested pod ports.
```